app/src/main/java/ch/hikemate/app/model/route/ListOfHikeRoutesViewModel.kt

```diff
@@ -82,4 +82,27 @@ open class ListOfHikeRoutesViewModel(
   fun selectRoute(hikeRoute: HikeRoute) {
     selectedHikeRoute_.value = hikeRoute
   }
+
+  private suspend fun selectRouteByIdAsync(hikeId: String) {
+    withContext(dispatcher) {
+      hikeRoutesRepository.getRouteById(
+          routeId = hikeId,
+          onSuccess = { route -> selectedHikeRoute_.value = route },
+          onFailure = { exception ->
+            Log.d(LOG_TAG, "[selectRouteByIdAsync] Failed to get route: $exception")
```

Comment:
Question: Why do you use Log.d to log an error? Shouldn't it be Log.e?

Reply:
Woups, did not notice that, thanks for pointing it out